modules/apps/jaggery/emm/src/emm/modules/policy.js

```diff
@@ -85,10 +85,10 @@ policyModule = function () {
                 response.status = "success";
                 response.content = policyListToView;
                 return response;
-            },function(responsePayload){
+            }, function (responsePayload) {
                 var response = {};
                 response.status = "error";
-                if(responsePayload  == "Scope validation failed"){
+                if (responsePayload == "Scope validation failed") {
```

Comment:
checking if permission denied based on a string returned from a back-end is not appropriate.
better checking for a status code like 401 Unauthorized